pom.xml

```diff
@@ -2677,5 +2692,16 @@
                 </plugins>
             </build>
         </profile>
+        <profile>
+            <id>javac-release</id>
+            <activation>
+                <jdk>[11,)</jdk>
```

Comment:
What does this mean exactly - in plain english?

Reply:
_When building using JDK 11 or greater, pass `--release 8` to `javac` and set `--illegal-access=permit` when running tests._